src/solver/optimisation/opt_alloc_probleme_a_optimiser.cpp

```diff
@@ -51,6 +51,9 @@ using namespace Antares;
 #else
 #define SNPRINTF snprintf
 #endif
+// #TODO check mps var max size
+#define MAX_VAR_NAME 200
+#define MAX_CNT_NAME 200
```

Comment:
It would be better to allocate only what's needed

Reply:
ok then freeing (NomsDesContraintes[i]) has to be moved to the new allocation place.